project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.1
```

Comment:
[`1.0.2`](http://developer.lightbend.com/blog/2017-09-19-sbt-1-migration-status-and-1-0-2-hotfix/#sbt-hotfix-1-0-2) is available with other bugfixes

Reply:
Thanks, I'll update this tomorrow! Unfortunately, not all sbt-doge features were moved to SBT (https://github.com/sbt/sbt-doge/issues/18)... We only need the missing operator for Travis, though.